File: Exercise3.java
import java.util.Scanner;
public class Exercise3 {

    public static void main (String [] args) {
        Scanner userinput = new Scanner (System.in);
        System.out.println("Enter the first side of the triangle: ");
        int a = userinput.nextInt();
        System.out.println("Enter the second side of the triangle: ");
        int b = userinput.nextInt();
        System.out.println("Enter the third side of the triangle: ");
        int c = userinput.nextInt();
        double d = (a+b+c)*(b+c-a)*(a-b+c)*(a+b-c);
        double area = Math.sqrt(d)/4;
        System.out.printf("The area of the given triangle is: %.2f", area);

    }
}

File: Prepatory weeks/27_09_2016_Python/exercise_3.py
import math
a = input("What's the first side of the triangle? ")
a = int(a)
b = input("What's the second side of the triangle? ")
b=int(b)
c = input("What's the third side of the triangle? ")
c=int(c)
d = (a+b+c)*(b+c-a)*(a-b+c)*(a+b-c)
area = (math.sqrt(d))/4
print ("The area of the given triangle equals to", area)
